### DISCUSSION TITLE
Apps Not Seeing SD Card

### DISCUSSION THREAD

**Discussion by Tourma** [FossifyOrg/General-Discussion#377] [Q&A]
Hello all,

I recently had to downgrade back to my Moto G7 Power*.  The one upside is that it includes a Micro SD Card port.  However, both Fossify File Manager and Gallery, (both of the apps that I use,) cannot seem to see it.  "Internal" is as deep as I can get in File Manager, and I don't see anyway to get down to root(?) to see the SD Card.

Is there some setting or button I'm missing?  Do Motorolas just not like doing this?

*Apparently the Pixel 5a has a known defect where the battery and motherboard just straight up die.  First smartphone I've had brick on me.

**Comment by Phurnace** [comment]
I've come across this issue too while using Gallery and just thought I'd add some extra info.

When plugging in an external USB stick it is recognized by my tablet and a notification pops up.  When opening up Gallery no pictures/folders are displayed that are stored on the USB.  It is possible to add the USB drive root via the 'Included Folder' option in the settings but still no pics or folders are shown.

My USB stick also contains some video files and VLC Player picks them up and creates folders and thumbnails without issue.

Cx File Explorer can see and open every picture on the USB and an individual pic can then be opened in Gallery using the 'More->Open with->Gallery' dropdown.  It is then possible to swipe (and slideshow) through all the pics in the folder (however they are sorted backwards).

I'm not a technical person but to me this feels like an issue within the app rather than anything permissions related and I'm wondering if this is a bug or something that's by design.  Strangely other photo/gallery apps also don't seem to be able to see pics on the USB drive.

Hope this helps.

**Comment by D15A-Jai-61** [comment]
The "Internal" button itself is used to switch from internal storage to external ones like USB and SD card; It took me quite some time to figure it out.
 If you don't see a popup menu that lets you select between internal and SD card when you press the "Internal" button at the top left below the search bar, then maybe your phone doesn't detect it or doesn't give the app permission to use it.

**Comment by tervox** [comment]
Hi, I don't know how to use this site properly, but I have a suggestion for improvement. Would it be possible to make the background images have the colors of the image displayed? I think it's better than just black or gray.

**Reply by naveensingh** [reply]
> how to use this site properly

There are some pointers at https://github.com/FossifyOrg and https://github.com/FossifyOrg/General-Discussion.

**Comment by naveensingh** [comment]
> The one upside is that it includes a Micro SD Card port. However, both Fossify File Manager and Gallery, (both of the apps that I use,) cannot seem to see it.

If you are still facing this issue in Fossify Gallery, please file a bug report [here](https://github.com/FossifyOrg/Gallery/issues/new?template=bug_report.yml). Also, include whether explicitly including the SD Card using **Manage included folders** option works.

> "Internal" is as deep as I can get in File Manager, and I don't see anyway to get down to root(?) to see the SD Card.

As mentioned above, tapping "Internal" allows you to switch between storage volumes.
